packages/server/graphql/mutations/helpers/createTeamPrompt.ts

```diff
@@ -0,0 +1,34 @@
+import MeetingTeamPrompt from '../../../database/types/MeetingTeamPrompt'
+import generateUID from '../../../generateUID'
+import {MeetingTypeEnum} from '../../../postgres/types/Meeting'
+import createNewMeetingPhases from './createNewMeetingPhases'
+
+const createTeamPrompt = async (teamId, facilitatorId, r, dataLoader) => {
+  const meetingType: MeetingTypeEnum = 'teamPrompt'
+  const meetingCount = await r
+    .table('NewMeeting')
+    .getAll(teamId, {index: 'teamId'})
+    .filter({meetingType})
+    .count()
+    .default(0)
+    .run()
+  const meetingId = generateUID()
+  const phases = await createNewMeetingPhases(
+    facilitatorId,
+    teamId,
+    meetingId,
+    meetingCount,
+    meetingType,
+    dataLoader
+  )
+  return new MeetingTeamPrompt({
+    id: meetingId,
+    teamId,
+    meetingCount,
+    phases,
+    facilitatorUserId: facilitatorId,
+    meetingPrompt: 'What are you working on today? Stuck on anything?' // :TODO: (jmtaber129): Get this from meeting settings.
+  })
+}
```

Comment:
+1 instead of having a bunch of helper methods, I think it'd be nice to have one class `TeamPromptMeetingActions` or something like that, with static methods. it'd allow having a single place for every action related to this meeting.

For example
```
class TeamPromptMeetingActions {
   static start(...) {}
   static startRecurring(rrule) {}
   static end(...) {} 
}
```

Then, we'd just call `TeamPromptMeetingActions.start(...)` 

it doesn't necessarily have to be all static, if there are parts that'd require calling multiple actions, we could instantiate this class and make it easier to call different methods. not a big deal, it works just fine as is, but I think this would be a bit clearer.

Reply:
Hmm is this common in other parts of the codebase?  I'm not opposed to making this a little more object-oriented, but AFAIK most of our code is pretty functional, and I don't totally understand the benefits to doing it this way beyond putting things in the same file.

Reply:
then don't do this. this is just my personal preference not to put every function in a separate file, especially when they're related. navigating that kind of code requires less context switching and makes it easier to spot things that could be shared/simplified.